runtime/moonbase/src/lib.rs

```diff
@@ -366,6 +374,7 @@ impl WeightToFeePolynomial for LengthToFee {
 	}
 }
 
+#[cfg(not(feature = "runtime-benchmarks"))]
 impl pallet_transaction_payment::Config for Runtime {
```

Comment:
We want to benchmark the implementation used in production

Reply:
Working on a proper solution involving a new cherry-pick on `polkadot-sdk`.